src/v/cluster/rm_stm.cc

```diff
@@ -2761,7 +2777,6 @@ rm_stm::apply_snapshot(stm_snapshot_header hdr, iobuf&& tx_ss_buf) {
         _log_state.lru_idempotent_pids.push_back(it->second);
     }
 
-    _bootstrap_committed_offset = data.offset;
```

Comment:
Now that we have this fiber, think we can also remove _bootstrap_committed_offset in rm_stm::apply()?

```
ss::future<> rm_stm::apply(model::record_batch b) {
    if (unlikely(!_bootstrap_committed_offset)) {
        _bootstrap_committed_offset = _c->committed_offset();
    }
```

Reply:
nice catch